src/hooks/useGlobalState.tsx

```diff
@@ -23,3 +24,18 @@ export function withGlobalContext<P>(
 const useGlobalState = (): [GlobalState, Function] => useContext(GlobalStateContext) as [GlobalState, Function]
 
 export default useGlobalState
+
```

Comment:
can you move this out? 

In regards tot he functionality, I'm on the fence about adding it as both methods have their respective pros and cons. May be better not to add it for now.